kube/services/fence/fence-canary-service.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: fence-canary-service
+spec:
+  selector:
+    app: fence-canary
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80 
+      name: http
+    - protocol: TCP 
+      port: 443 
+      targetPort: 443
+      name: https
+  type: NodePort
```

Comment:
Make this 'ClusterIP' - we don't need a reserved port - it's hard to change existing services though :-p